gui/mozregui/report.py

```diff
@@ -60,11 +61,33 @@ def status_text(self):
         return msg % self.state_text
 
 
+def bulk_action_slots(slots, signal_object, slot_object, action):
+    if signal_object and slot_object:
```

Comment:
I would remove the if here. callers have to know how to call this - this is in internal function. Plus I would prefer this to fail instead of silently doing nothing in that case.